examples/biobert_ner/bio_ner_predictor.py

```diff
@@ -165,32 +190,36 @@ def pack(self, data_pack: DataPack,
         ner to the original subwords and convert predictions to something that
         makes sense in a word-by-word segmentation
         """
-
+        # print(list(data_pack.get(Sentence)))
         if output_dict is None:
             return
 
         for i in range(len(output_dict["Subword"]["tid"])):
             tids = output_dict["Subword"]["tid"][i]
             labels = output_dict["Subword"]["ner"][i]
 
-            # Filter to labels not in `self.ignore_labels`
+            entities = []
+            # Filter to labels not in `self.ft_configs.ignore_labels`
             entities = [dict(idx=idx, label=label, tid=tid)
                         for idx, (label, tid) in enumerate(zip(labels, tids))
                         if label not in self.ft_configs.ignore_labels]
 
-            entity_groups = self._group_entities(entities, data_pack, tids)
+            entity_groups = self._compose_entities(entities, data_pack, tids)
             # Add NER tags and create EntityMention ontologies.
-            for first_tid, last_tid in entity_groups:
+            for first_idx, last_idx in entity_groups:
                 first_token: Subword = data_pack.get_entry(  # type: ignore
-                    first_tid)
+                    tids[first_idx])
                 first_token.ner = 'B-' + self.ft_configs.ner_type
+                begin = first_token.span.begin
 
-                for tid in range(first_tid + 1, last_tid + 1):
+                for tid in tids[(first_idx + 1):(last_idx + 1)]:
                     token: Subword = data_pack.get_entry(tid)  # type: ignore
                     token.ner = 'I-' + self.ft_configs.ner_type
```

Comment:
It is actually not important to add `ner` to the tokens. We can actually save some space by not setting them.

Reply:
Sounds good

Reply:
Sounds good